website/content/docs/agent/telemetry.mdx

```diff
@@ -542,6 +542,8 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.grpc.server.streams`                        | Measures the number of active gRPC streams handled by the server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | streams                           | gauge   |
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | streams                           | gauge   |
+| `consul.xds.server.idealSessionsMax`                | The ideal maximum number of xDS sessions per server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | sessions                          | gauge   |
```

Comment:
Great point! It's the server's opinion of the number of xDS sessions it should handle to achieve an even spread of load across the cluster.

It is used as a hard limit, but might take a while to be enforced because the server will slowly drain excess sessions, to avoid causing major disruption by draining them all at once.

I think the reason we chose to include "ideal" in the metric name is to prevent folks from getting confused by seeing that a server's "maximum" number of sessions is lower than its actual number of sessions.

Do you think we should drop "ideal" from the name too? Or maybe have a clearer name?